packages/clay-card/src/Detail.tsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import React from 'react';
+
+import classNames from 'classnames';
+
+import Context from './Context';
+
+interface CardDetailProps extends React.HTMLAttributes<HTMLDivElement> {
+	children: React.ReactNode;
+}
+
+export type CardDetailType = React.FunctionComponent<CardDetailProps>;
+
+export const Detail: CardDetailType = ({
+	children,
+	className,
+	...otherProps
+}) => {
+	const {displayType} = React.useContext(Context);
+	const isCardType = {
+		horizontalInteractive: displayType === 'horizontal interactive',
+		interactive: displayType === 'interactive',
+	};
+
+	const TagName =
+		isCardType.interactive || isCardType.horizontalInteractive
```

Comment:
This could be:

```suggestion
		displayType === 'interactive' || displayType === 'horizontal interactive'
```

Then we can remove the `isCardType` from here.